WTWSMS/events/VIET_WTWSMS_events.txt

```diff
@@ -0,0 +1,1544 @@
+				10 = { add_random_education_trait = intrigue }
+			}
+			#Personality Archetypes
+			random_list = {
+				20 = { # Sassy girl
+					set_character_flag = VIET_corduenian_sassy
+					add_trait = cynical
+					add_trait = erudite
+					add_trait = stubborn
+					add_trait = honest
+				}
+				20 = { # Social climber
+					set_character_flag = VIET_corduenian_ambitions
+					add_trait = greedy
+					add_trait = diligent
+					add_trait = ambitious
+					add_trait = gregarious
+				}
+				20 = { # Wallflower
+					set_character_flag = VIET_corduenian_wallflower
+					add_trait = shy
+					add_trait = humble
+					add_trait = slothful
+					add_trait = kind
+				}
+				20 = { # tomboy
+					set_character_flag = VIET_corduenian_tomboy
+					add_trait = brave
+					add_trait = gregarious
+					add_trait = wroth
+					add_trait = diligent
+				}
+				20 = { # religious
+					set_character_flag = VIET_corduenian_pious
+					add_trait = zealous
+					add_trait = temperate
+					add_trait = charitable
+					add_trait = chaste
+				}
+				20 = { # asshole
+					set_character_flag = VIET_corduenian_asshole
+					add_trait = cruel
+					add_trait = envious
+					add_trait = arbitrary
+					add_trait = proud
+				}
+				20 = { # wolf in sheep's clothing
+					set_character_flag = VIET_corduenian_wolfsheep
+					add_trait = deceitful
+					add_trait = cynical
+					add_trait = gregarious
+					add_trait = patient
+				}
+				20 = { # wholesome good girl
+					set_character_flag = VIET_corduenian_wholesome
+					add_trait = kind
+					add_trait = trusting
+					add_trait = humble
+					add_trait = content
+				}
+				20 = { # nerd
+					set_character_flag = VIET_corduenian_nerd
+					add_trait = kind
+					add_trait = erudite
+					add_trait = just
+					add_trait = shy
+				}
+				20 = { # People person
+					set_character_flag = VIET_corduenian_personable
+					add_trait = kind
+					add_trait = gregarious
+					add_trait = charitable
+					add_trait = gluttonous
+				}
+			}
+		}
+	}	
+	option = { # Waste of time
+		name = VIETmisc.4013.b
+		piety = 5
+		if = {
+			limit = {
+				trait = lustful
+			}
+			random = {
+				remove_trait = lustful
+				chance = 45
+			}
+		}
+	}
+	option = { # Study the stories and legends
+		name = VIETmisc.4013.c
+		prestige = 5
+		random_list = {
+			10 = { add_trait = erudite }
+			10 = { change_learning = 1 }
+			70 = { }
+		}
+	}
+	option = { # What's so special?
+		name = VIETmisc.4013.d
+		add_character_modifier = {
+			name = VIET_skeptical_ruler
+			duration = 365
+		}
+	}
+}
+
+
+# Come across a Corduenian ruin
+character_event = {
+	id = VIETmisc.4014
+	desc = VIETmisc.4014.desc
+	picture = GFX_evt_ancient_cairn
+	
+	is_triggered_only = yes
+	
+	min_age = 16
+	only_rulers = yes
+	capable_only = yes
+	prisoner = no
+	#war = no
+	#culture = corduenian
+	
+	trigger = {
+		in_command = no		
+		any_demesne_province = {
+			#Where Corduenians are in the start of the WTWSMS game
+			province_id = 683
```

Comment:
I think these province_id conditions should be inside 'OR'.

Reply:
Oh yeah, that's true. Good catch!